Extract Obsolete Operating Systems from LDAP

Shad0wC0ntr0ller opened the pull request:
Simple module to extract obsolete operating systems from LDAP.

KenjiEndo15 commented:
Cool stuff :+1: 
I've seen in your code that you consider Windows Server 2012 as obsolete, but both 2012 and 2012 R2 will end in [October 10, 2023](https://learn.microsoft.com/en-us/lifecycle/announcements/windows-server-2012-r2-end-of-support). I wouldn't consider that as obsolete (yet).

Shad0wC0ntr0ller commented:
You are absolutely right, I meant to remove that, I have updated the code to remove Server 2012. Additionally I have added a section to save all obsolete hostnames to a file in the /tmp/ directory. Please let me know if there is anything else you would like changed or updated.

Shad0wC0ntr0ller commented:
Sorry im a github n00b
I wanted to make a new pull request for the file that got added to this one. The file is called trust.py and will extract the available trusts and show the direction.

If you could please advise me on what to do I don't want to just keep playing around with settings. thank you

PS - also working on a new module that implements the raiseChild.py into CrackMap which should be ready soon.

![trust](https://user-images.githubusercontent.com/90877534/222935415-7d38c3e8-9b1c-45b8-8b42-47354cb1bf9f.png)
